index.html

```diff
@@ -93,38 +93,148 @@
       cursor: pointer;
       transition: background 200ms linear;
     }
+
+    input.search {
+      color: black;
+      border: 1px dashed #555;
+      margin: 1rem auto;
+      display: flex;
+      width: 100%;
+      max-width: 1200px;
+      font-size: 1rem;
+      line-height: 1.5;
+      border-radius: 1.5rem;
+      padding: 0.5rem;
+      padding-left: 40px;
+      background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16"><path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/></svg>');
+      background-repeat: no-repeat;
+      background-position: 16px center;
+      background-size: 16px 16px;
+    }
   </style>
 
   <script>
-    const symbols = [
-      "©", "®", "™", "“", "”", "£", "…", "½", "¼", "∞", "é", "á", "à", "ç", "€",
```

Comment:
Moving toward one line per character might also help reduce merge conflicts in the future (or at least make them easier to sort out) 🫠